woocommerce/woocommerce-android
# Resize filter fragment into dialog in tablet

Closes: #10571 


### Description

This PR modifies the filters fragment to display as a dialog on tablet devices, maintaining visibility of the underlying context and ensuring an optimal UI appearance. Conversely, on phones, the filters fragment should continue to operate in a full-screen mode.

### Testing instructions

1. Open order listing screen on tablet device
2. Select filters in the order listing screen
3. Ensure you see a dialog with filter categories
4. On selecting any category, ensure you see another dialog to select the filter options
5. Run through the same steps on mobile phone and ensure the filters screen appears as full screen on phones.

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/1331230/ee6e8329-3ea2-4bbe-815b-330b1aabec80
